docs/Installation/Re-Installing_Zenko.rst

```diff
@@ -0,0 +1,113 @@
+
+###################
+Re-Installing Zenko
+###################
+
+If you are installing Zenko on a fresh cluster, the instructions provided in
+the `Installation Guide <./installation_guide.html>`_ are sufficient. If, however,
+you are reinstalling Zenko on an existing cluster (while assessing or testing
+Zenko, for example), you must perform the following additional steps.
+
+Destroy All Volumes
++++++++++++++++++++
+Any Kubernetes cluster deloyed for Zenko must have a logical volume associated
+with each node. If you reinstall Zenko onto a prior installation's node
+structure, these existing volumes will prevent successful deployment.
+
+Before reinstalling Zenko, destroy all volumes in the cluster. Do not reuse the
+existing volumes.
+
+Diagnostic
+----------
+
+A Scality engineer used the following diagnostic to determine the source of
```

Comment:
What issue is this helping Diagnose? Also cannot be done if all the volumes have been deleted.

Reply:
Bottom line was it was the only technical information I could get my hands around, and figured a "bad" log reading would be helpful in identifying the problem (manifests as duplicate node ID numbers). I am fine if this section is cut.